.pre-commit-config.yaml

```
@@ -31,20 +38,23 @@ repos:
+        args: ['--config', '.taplo.toml']
       - id: pytest
         name: Run pytest
         entry: pytest
         language: system
         types: [python]
         pass_filenames: false # Run full test suite
         stages: [pre-commit] # Only run on pre-commit
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v4.0.0-alpha.8
+    hooks:
+      - id: prettier
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)